.github/workflows/prod-sync-odfe-cli.yml

```diff
@@ -0,0 +1,29 @@
+name: Prod Sync ODFE CLI Artifacts
+
+on:
+  repository_dispatch:
+    types: [prod-sync-odfe-cli]
+
+jobs:
+  sync-odfe-cli-artifacts-prod:
+    name: sync-odfe-cli-artifacts-prod
+    runs-on: [ubuntu-18.04]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+    - name: Required Packages
+      run: ./release-tools/scripts/required_packages.sh
+    - name: S3 sync artifacts
+      run: |
+        #!/bin/bash
+        S3_BUCKET_PROD="artifacts.opendistroforelasticsearch.amazon.com"
+        S3_BUCKET_PLUGIN="staging.artifacts.opendistroforelasticsearch.amazon.com"
```

Comment:
Maybe rename it to `S3_BUCKET_STAGING` since it is not plugin bucket anymore

Reply:
Sure